src/cs50.h

```diff
@@ -120,6 +120,11 @@ typedef char *string;
 #undef WARN
 #define WARN(func) func
 
+// If compiler isn't GCC or Clang it may not support __extension__, better to
+// suppress this error and hope for the best
```

Comment:
I mean to be completely honest, in my own code I almost never use `/**/`. Capitalization-wise I tend to capitalize complete sentences. I'll change it though!